tools/ctl/windows/aws-otel-collector-ctl.ps1

```diff
@@ -53,7 +53,7 @@ $UsageString = @"
 "@
 
 $AOCServiceName = 'AWSOTelCollector'
-$AOCServiceDisplayName = 'AWS OTel Collector'
+$AOCServiceDisplayName = 'ADOT Collector'
 $AOCDirectory = 'Amazon\AWSOTelCollector'
```

Comment:
Does The directory name need to change also?

Reply:
It would be a breaking change, we shall track through an isse.